content/knitr/options.md

```diff
@@ -9,9 +9,9 @@ show_toc: true
 The **knitr** package has provided a lot of chunk options for customizing nearly
 all components of code chunks, such as the source code, text output, plots, and
 the language of the chunk, etc. It has also offered some options at the package
-level to customize the knitting process. This page documents all chunk options
-and package options available in **knitr**. The default values of these options
-are in parentheses in the list items.
+level to customize the knitting process. This page documents all chunk options 
+(that apply across all language engines) and package options available in **knitr**. 
```

Comment:
I'm not sure if it's worth highlighting the fact that these chunk options are engine-agnostic. I'm afraid that it is too technical for the short introduction here. On other other hand, these options are not truly engine-agnostic, e.g., `dev` is mainly for R (the supported graphical devices are in R).

Reply:
Makes sense. Put it back to how it was!